suite-common/suite-types/src/route.ts

```diff
@@ -6,8 +6,9 @@ type RouteKeys =
     | 'params'
     | 'exact'
     | 'isForegroundApp'
+    | 'isNestedRoute'
```

Comment:
is this used somehow? or does it have just informational meaning?

Reply:
It's necessary for the type where PageName is extracted from Route type. 

```js
export type PageName = Exclude<Route, { isForegroundApp: true } | { isNestedRoute: true }>['name'];
```

Nested Route is excluded because it doesn't have it's own page. Otherwise I'd have to assign some component to the route, and it's exactly what we want to avoid with nested components, because they get a component assigned inside it's parent route. 

For more context you can also check Router.tsx, where it's being mapped. 

If i remove that exclusion of isNestedRoute, typescript complains:

![image](https://github.com/user-attachments/assets/801ea129-a633-40d0-9f33-27f8b9c78cfe)

Reply:
ok makes sense, thanks